rfc/php_technical_committee.wiki

```diff
@@ -0,0 +1,183 @@
+====== PHP RFC: PHP Technical Committee ======
+  * Version: 0.3
+  * Date: 2023-03-03
+  * Author: Jakub Zelenka (bukka@php.net), Larry Garfield (larry@garfieldtech.com)
+  * Status: Draft
+
+===== Introduction =====
+
+PHP uses its voting system to decide about changes to PHP. This was created mainly for user facing changes. It works
+well for that purpose but it is not that effective for deciding about purely technical changes that impact PHP
+internals and extension API's. It is not clear how it should resolve conflicts on the technical basis between
+contributors that sometimes happen.
+
+
+===== Proposal =====
+
+This RFC proposes an introduction of the PHP Technical Committee (TC) that would decide about all technical aspects of
+php-src when there is a disagreements between core developers.
+
+==== Definitions ====
+
+The text contains various terms that need further clarification to prevent future wording dispute. Here are their
+definitions:
+
+  * php-src - Source code in https://github.com/php/php-src/.
+  * contributor - Anyone who has already contributed to php-src.
+  * core developer - Anyone with write commit access to php-src.
+  * dev branch - php-src branch consisting of PHP- prefix and major and minor version number (e.g. PHP-8.2).
+  * maintainer - person responsible for core subsystem (e.g. extension or SAPI) that is listed in the php-src EXTENSION file or the php-src OWNERS files if added in the future.
+  * issue - php-src GitHub issue, including pull requests.
+  * process changes - Any changes impacting the way how PHP project is managed.
+  * pull request - php-src GitHub pull request.
+  * RM - PHP release managers.
+  * user facing changes - Changes in PHP language and its semantics, user visible functions, classes, interfaces, constants and other user visible constructs.
+  * technical change - Any change to php-src that has no user-facing change, including the implementation details of an RFC-accepted user facing change.
+  * technical aspect - The portions of  pull request or commit in php-src or commit to php-src master and dev branches that would be technical changes.
+  * technical decision - Decision on a technical aspect.
+  * TC - PHP Technical Committee.
```

Comment:
AWOL should be added here.

As I have no idea what it meant before looking it up.

Reply:
It was added by @Crell . I assume it is `absent without leave`. Is that right? Maybe we should replace it in the text with something that everyone will understand or add the definition?

Reply:
The questions are for @Crell btw. :smile:

Reply:
Yes, let's replace it with something more formal/contextually appropriate.

Reply:
@Crell Any suggestion?

Reply:
See edit suggestions elsewhere in the thread.